### ISSUE TITLE
ROS Noetic Distro

### ISSUE THREAD

**Issue by jacobsayono** [dusty-nv/ros_deep_learning#119]
Will ROS Noetic packages and builds work with this?

**Comment by dusty-nv** [comment]
@jacobsayono yes, and this package is built into the `dustynv/ros:noetic-pytorch-l4t-*` containers from https://github.com/dusty-nv/jetson-containers
